### PR TITLE
MAM-3095-load-test-20k-ar-reduce

### PR DESCRIPTION
**Pull request by jtomchak**
* reduce / eliminate UpdateForYou from making repeated active record calls. grouping them once by the already running channel update worker and using those.

Safari 2023-11-15 at 11 43 22@2x
Safari 2023-11-15 at 11 44 22@2x

### CODE REVIEW

File: app/lib/mammoth/channels.rb [hunk 1]
```diff
@@ -64,7 +68,7 @@ def account_ids(accounts)
     # Channel includes id, title, description, owner
     def list(include_accounts: false)
       cache_key = include_accounts ? 'channels:list:w_accounts' : 'channels:list'
-      Rails.cache.fetch(cache_key, expires_in: 60.seconds) do
+      Rails.cache.fetch(cache_key, expires_in: 1.hour) do
```
Comment (rileyhCode) [inline app/lib/mammoth/channels.rb:71]: Just confirming this change from 1 min to 1 hour is expected.

File: app/lib/mammoth/channels.rb [hunk 2]
```diff
@@ -75,7 +79,7 @@ def list(include_accounts: false)
     # GET channel by id and return all details
     def find(id)
       cache_key = "channels:list:#{id}"
-      Rails.cache.fetch(cache_key, expires_in: 60.seconds) do
+      Rails.cache.fetch(cache_key, expires_in: 1.hour) do
```
Comment (rileyhCode) [inline app/lib/mammoth/channels.rb:82]: Just confirming this change from 1 min to 1 hour is expected.

**Review by rileyhCode — APPROVED** [review]
Pls review the comments about interval change, just to confirm they are expected. LGTM otherwise.

File: app/lib/mammoth/channels.rb (hunk 1 above)
Comment (jtomchak) [reply app/lib/mammoth/channels.rb:71]: yup. I wanted to keep the channels and the accounts that we've added to them cached, but when they are edited not have to wait more than an hour for them to populate

File: app/lib/mammoth/channels.rb (hunk 2 above)
Comment (jtomchak) [reply app/lib/mammoth/channels.rb:82]: same here. channel details should be good from hour to hour

**Pull request merged**